### ISSUE TITLE
Video playback refreshing every 30 seconds

### ISSUE THREAD

**Issue by Jman012** [Jman012/Wasserflug-tvOS#1]
When playing a video from a creator's tab (not the Home tab), the video will refresh every 30 seconds. It resumes from where it left off, but is met with a black screen and a loading indicator for half a second. This is not good for user experience of watching a long video where every 30 seconds it is interrupted.

This affects version 1.0.5.

**Comment by Jman012** [comment]
Possible cause: https://github.com/Jman012/Wasserflug-tvOS/commit/7d80703afb30939ef33e8302ea22595c1687b12f#diff-563ce2de4c5989fb92bfab1315c7c9365d8b2d22449e066b52a8a570663f42f8R122

That, or this commented-out code means it was never working: https://github.com/Jman012/Wasserflug-tvOS/commit/da3521c7b1c7ffbddee7ff0412fc9dd0dc2eb707#diff-563ce2de4c5989fb92bfab1315c7c9365d8b2d22449e066b52a8a570663f42f8L79

I seem to remember having problems with getting the timer to invalidate on the view model correctly. The root cause is that the timer runs every 30 seconds which does a small update to the `CreatorContentView` (the Livestream button's appearance), which causes a view update that also happens to refresh the view for the video player. I think that's because the video player always updates the `AVPlayer`/`AVPlayerItem` on a SwiftUI update invocation. Last time I tried to solve this, I tried to stop the livestream 30s update when watching a video. Perhaps I should see about making the video player update smarter and not update the `AVPlayerItem` when it isn't needed.
